library/core/src/num/int_macros.rs

```diff
@@ -898,6 +898,29 @@ macro_rules! int_impl {
             acc.checked_mul(base)
         }
 
+        /// Returns the square root of the number, rounded down.
+        ///
+        /// Returns `None` if `self` is negative.
+        ///
+        /// # Examples
+        ///
+        /// Basic usage:
+        /// ```
+        #[doc = concat!("assert_eq!(10", stringify!($SelfT), ".checked_isqrt(), Some(3));")]
+        /// ```
+        #[stable(feature = "isqrt", since = "1.73.0")]
+        #[rustc_const_stable(feature = "isqrt", since = "1.73.0")]
```

Comment:
New methods should be unstable by default, so this should be
```suggestion
        #[unstable(feature = "isqrt", issue = "none")]
        #[rustc_const_unstable(feature = "isqrt", issue = "none")]
```
for now. After you open a tracking issue, its number should be put here.